testutil/promrated/promrated.go

```diff
@@ -119,3 +118,15 @@ func reportMetrics(ctx context.Context, config Config) {
 		networkEffectiveness.With(networkLabels).Set(stats.ValidatorEffectiveness)
 	}
 }
+
+func contains(arr []string, s string) bool {
```

Comment:
add godoc. Or, rather can remove this helper as it is only used once

Reply:
I opted for godoc as I think it's still cleaner to have this extracted 👍🏼